src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -46,7 +46,7 @@ public async Task<int> ExecuteCommandAsync(VerifyArgs verifyArgs)
                         fingerprint,
                         _defaultFingerprintAlgorithm)).ToList();
 
-                var verifierSettings = SignedPackageVerifierSettings.GetVerifyCommandDefaultPolicy(clientAllowListEntries: allowListEntries);
+                var verifierSettings = SignedPackageVerifierSettings.GetVerifyCommandDefaultPolicy(revocationMode: verifyArgs.RevocationMode, clientAllowListEntries: allowListEntries);
```

Comment:
`verifyArgs.RevocationMode` should you check for null?

Reply:
`verifyArgs.RevocationMode` should never be null... it defaults to online